api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -112,5 +116,15 @@ def update_company_active_subscription(
 
         return response
 
+    def _get_or_create_company_by_domain(self, domain: str) -> dict:
+        # try to find the organisation via it's domain
+        company = self.client.get_company_by_domain(domain)
+        if not company:
+            # otherwise create a new organisation with the domain
+            # as the name.
+            company = self.client.create_company(name=domain)
```

Comment:
I'm not really sure what benefit the comments here bring, since it's all self-documented by the method names.

Reply:
Lol, yep, very good point. I've become everything I hate in a developer 😂